src/django/contricleaner/lib/serializers/row_serializers/row_clean_field_serializer.py

```diff
@@ -8,7 +8,9 @@ def __init__(self, field: str, new_field: str) -> None:
         self.new_field = new_field
 
     def validate(self, row: dict, current: dict) -> dict:
-        clean_value = clean(row[self.field])
+        field = row.get(self.field, '')
```

Comment:
It can't be reverted completely, because `clean` is supposed to receive a value. In the case when a user doesn't add any name or address, the `clean` function will return an error that does not follow the expectations. 
I propose this variant of the solution `clean_value = clean(row.get(self.field, ''))`